client/src/pages/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -33,6 +35,17 @@ interface Props {
 
 const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
   const classes = useStyles();
+  const { isOpen, setIsOpen, currentStep, setCurrentStep } = useTour();
```

Comment:
Everything in the signup form is self explanatory as well, I don't we need a tour bubble here :)

Reply:
Yes, I agree. I chose signup form because I didn't want to work on some logged-in components that may disturb the constructions of my teammates.